RULES.md

```diff
@@ -118,7 +118,7 @@ Additional details regarding the notices' context is provided in [`NOTICES.md`](
 | [`AttributionWithoutRoleNotice`](#AttributionWithoutRoleNotice)                   	| Attribution with no role.                                                                                                                                   	|
 | [`DuplicateRouteNameNotice`](#DuplicateRouteNameNotice)                           	| Duplicate  `routes.route_long_name`. Duplicate `routes.route_short_name`. Duplicate combination of fields `route_long_name`  and `routes.route_short_name`. 	|
 | [`EmptyColumnNameNotice`](#EmptyColumnNameNotice)                                 	| A column name is empty.                                                                                                                                     	|
-| [`EmptyRowNotice`](#EmptyRowNotice)                                               	| A file is unknown.                                                                                                                                          	|
+| [`EmptyRowNotice`](#EmptyRowNotice)                                               	| A row in the input file has only spaces.                                                                                                                                          	|
```

Comment:
Definitely ot a big deal but I usually copy-paste the markdown in an external tool to ensure the raw `RULES.md` keeps looking good (with all "|" characters in tables aligned). You can use the following https://www.tablesgenerator.com/markdown_tables to do so.

Reply:
Interesting! Will do

Reply:
It's done, I also fixed the typos in NOTICES.md, as mentioned in https://github.com/MobilityData/gtfs-validator/issues/943